sections/pyenv.zsh

```diff
@@ -13,21 +13,27 @@ SPACESHIP_PYENV_PREFIX="${SPACESHIP_PYENV_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREF
 SPACESHIP_PYENV_SUFFIX="${SPACESHIP_PYENV_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
 SPACESHIP_PYENV_SYMBOL="${SPACESHIP_PYENV_SYMBOL="🐍 "}"
 SPACESHIP_PYENV_COLOR="${SPACESHIP_PYENV_COLOR="yellow"}"
+SPACESHIP_PYENV_USE_VENV="${SPACESHIP_PYENV_USE_VENV=false}"
 
 # ------------------------------------------------------------------------------
 # Section
 # ------------------------------------------------------------------------------
 
 # Show current version of pyenv Python, including system.
 spaceship_pyenv() {
-  [[ $SPACESHIP_PYENV_SHOW == false ]] && return
+  # Do nothing if SHOW is false or pyenv is not installed
+  [[ $SPACESHIP_PYENV_SHOW == false ]] && spaceship::exists pyenv && return
 
   # Show pyenv python version only for Python-specific folders
   [[ -f requirements.txt ]] || [[ -n *.py(#qN^/) ]] || return
 
-  spaceship::exists pyenv || return # Do nothing if pyenv is not installed
```

Comment:
Please don't make changes irrelevant to the PR. This flow of conditionals are styles followed across code base.

Reply:
Right, my bad, I thought that this check after checking for python files was an error. I'll fix.